_download-meta/openttd-nightlies.md

```diff
@@ -0,0 +1,10 @@
+---
+folder: nightlies/trunk
+---
+
+For OpenTTD you can use the original Transport Tycoon Deluxe data files (you need to own a Transport Tycoon Deluxe CD).
+There are also the free alternatives: [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx), [OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx).
+These can be installed automatically by the Windows and OS/2 installers.
+Please refer to the [readme](https://binaries.openttd.org/releases/1.8.0/readme.txt) for more information.
```

Comment:
This is hardcoded to the 1.8.0 release. Is this just due to it being readme.md in master (and will be updated when 1.9 is released), or an oversight?